requirements-docker.txt

```diff
@@ -0,0 +1,12 @@
+# Base requirements for ssm
+
+argo-ams-library
+pyopenssl
+cryptography
+stomp.py
+python-daemon
+python-ldap
```

Comment:
I don't think this would work in the rocky9. Please correct me if I am wrong. We should change this to python3-ldap.